tests/host_tools/logging.py

```diff
@@ -11,32 +11,32 @@
 from threading import Thread
 
 
-class Fifo:
-    """Facility for creating and working with named pipes (FIFOs)."""
+class File:
```

Comment:
Why this change? Why not keep using fifos? I think we should stick to fifos because in the long run we'll add high-density tests. Fifos at least have limited capacity, so, if nobody's reading from the other end, they won't accept new incoming data. Files on the other hand will fill up the disk if anything's wrong. If there's a synchronization or I/O issue with fifos, please fix that instead.

Reply:
Disclaimer: I'm fine with both files or fifos here.

There are pros and cons to each, but just to make sure we care about the right ones, will this code ever be used for high-density testing (honest question)?

Reply:
I'd see it that way, yup. I could see the use of throughput metrics in high density, as well as the boot time log (which we looked for in the `test_multiple_microvms` scenarios). So unless we rewrite the whole test harness (or write a new one for perf&stress), I think this framework might be used in density tests too.